packages/cfg-core/package.json

```diff
@@ -27,7 +27,6 @@
     "url": "git+https://github.com/syntest-framework/syntest-core.git"
   },
   "scripts": {
-    "build": "npm run build:compile",
```

Comment:
Why do we remove the `build`  and `prepare` commands?

Reply:
Because lerna uses these standard scripts and it breaks the build order if we fill something in here.